frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -480,7 +480,7 @@ export const funnelLogic = kea<funnelLogicType>({
             personsModalLogic.actions.loadPeople({
                 action: { id: step.action_id, name: step.name, properties: [], type: step.type },
                 breakdown_value: breakdown_value || '',
-                label: `Persons who completed Step #${stepNumber} - "${step.name}"`,
+                label: `Persons who dropped off at Step #${stepNumber} - "${step.name}"`,
```

Comment:
Thanks for creating this PR! The other issue with this was that stepNumber was a negative number, so maybe replace that with `${Math.abs(stepNumber)}`?